crates/starknet-devnet/tests/get_transaction_by_hash.rs

```diff
@@ -64,7 +64,7 @@ mod get_transaction_by_hash_integration_tests {
             starknet_rs_core::types::DeclareTransaction::V1(declare_v1),
         ) = result
         {
-            let expected = "0x05f47de31f16e7a474f6279c02b1783a86905c12bf0c2978bc2893e9d4f071a8";
```

Comment:
Why did applying USC change the expected hash?

Reply:
it didn't, it came from the blockifier update. This branch is rebased on new version of main